docs/sources/reference/builder.md

```diff
@@ -650,6 +672,28 @@ The copy obeys the following rules:
 - If `<dest>` doesn't exist, it is created along with all missing directories
   in its path.
 
+### Set ownership of content added by COPY
+
+By default, all new files and directories added by the `COPY` instruction are
+created with a UID and GID of 0. To copy files with a different owner, specify
+a user or uid using this form:
+
+    COPY --user=<uid_or_username> <src>... <dest>
```

Comment:
I know this came up in the original discussion, but wasn't discussed in-depth, but should we implement `uid:gid` and `username:groupname` as well?

Should they be separate flags? `--user=...` `--group=...`?